etc/init.d/kano-settings

```diff
@@ -10,11 +10,13 @@
 ### END INIT INFO
 
 . /lib/lsb/init-functions
+. gettext.sh
```

Comment:
should this have an absolute path, given it's pulled from a former sysv init script?